mmv1/templates/terraform/custom_flatten/compute_network_numeric_id.go.erb

```diff
@@ -0,0 +1,5 @@
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+    numeric_id := d.Get("id")
```

Comment:
> One way to work around this is to add a `"numericId"` field to the `res`/response variable being used to set values in state. Does that make sense?

Took a bit but I understand what you mean by this now. I ended up adding `numericId` into the decoder where it grabs `res["id"]` before it's overwritten by the `d.SetId()` function. Let me know if this is a good alternative or if I misunderstood your comment. To me this is not relying on Terraform since we are grabbing the resulting request `id` and using it to write it in the `numeric_id` attribute

https://github.com/GoogleCloudPlatform/magic-modules/blob/797ee95d6c8c4c81995743afd8f9f2a7a0edc047/mmv1/templates/terraform/decoders/compute_network.go.erb#L1